docs/ecosystem-roadmap.md

```diff
@@ -0,0 +1,80 @@
+
+### Standard Handles
+
+The standard handles are a series of handles, `STDIN`, `STDOUT`, and `STDERR`, that are introduced a part of process space on startup and represent a place where information can be accepted on the way in and sent back on the way out (including a special place to report errors out). For command-line applications, these must always exist when the application starts and are either inherited from the parent automatically, set explicitly by the parent, or created automatically by the operating system if neither of the first two are specified/permitted. For classic windows applications, these may be blank on startup, but can be implicitly or explicitly inherited from the parent or allocated, attached, and freed during runtime by the application itself.
+
+Standard handles do not imply a specific type of attached device, though in the case of command-line applications, the device is most commonly a console device, file (from redirection in a shell), or a pipe (from a shell connecting the output of one utility to the input of the next). It may also be a socket or any other type of device.
+
+### TTY/PTY
+
+On non-Windows platforms, the TTY and PTY devices represent respectively either a true physical device or a software-created pseudo-device that represent the same as a Windows console session: a channel where communication between a command-line client application and a server host interactivity application or physical keyboard/display device can exchange text-based information.
+
+### Clients and Servers
+
+Within this space, we're referring to "clients" as applications that do the work of processing information and running commands. The "server" applications are those that are responsible for the user interface and are workers to translate input and output into standard forms on behalf of the clients.
+
+### Console Subsystem
+
+This is a catch-all term representing all modules in the console team's space. It specifically refers to a flag that is a part of the Portable Executable header that specifies whether the starting application is either a command-line/console application (and must have standard handles to start) or a windows application (and does not need them).

```

Comment:
hmm... console team doesn't mean much to external folks

Reply:
clarified to not talk about us